yellowbox/extras/logstash.py

```diff
@@ -237,8 +259,10 @@ def disconnect(self, network: Any):
         """Does nothing. Conforms to YellowService interface."""
         pass
 
-    def _filter_records(self, level: int) -> Iterable[Dict[str, Any]]:
+    def filter_records(self, level: Union[str, int]) -> Iterator[Dict[str, Any]]:
         """Filter records in the given level or above."""
+        level = _level_to_int(level)
+
         return (record for record in self.records if
                 logging.getLevelName(record["level"]) >= level)
```

Comment:
is it possible for a record to arrive without a level?

Reply:
No

Reply:
don't your own tests create these scenarios?

Reply:
It shouldn't be a thing anyway, it means that the protocol isn't all screwed up but the messages sent contain non-protocol data.

Reply:
You don't want to skip it, you want to throw an error for sure, all I'm saying is that maybe, just maybe we can document it throwing KeyError. Don't think there's a better error to send anyway